module.js

```diff
@@ -15,6 +15,7 @@ var hook = function(req, res) {
 
 var github = function(payload, localToken) {
     // TODO: Only acknowledge pushes to the "Master" branch.
+    console.log("Repo: "+payload.repository.full_name);
```

Comment:
Ah, makes sense. No point in running it against the whole file again. I completely agree that we would be better to clean up code across the board. Maybe we could have beautify run as a pre-commit hook.

In the meantime, we can add spaces around the `+` in this line.